content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -242,19 +242,21 @@ When a Pod's containers are Ready but at least one custom condition is missing o
 
 ### Pod network readiness {#pod-has-network}
```

Comment:
@sftim wdyt about changing the name of this section? This was the name of the old condition and we renamed it to PodReadyToStartContainers.

Reply:
How about “Pod sandbox readiness”, then?

Reply:
We can make that change after this PR lands.